docs/package.json

```diff
@@ -34,7 +32,8 @@
     "start": "hugo server"
   },
   "dependencies": {
-    "@tylerbu/custom-api-documenter": "~0.3.4-7",
+    "@fluid-tools/api-markdown-documenter": "file:../tools/api-markdown-documenter",
```

Comment:
Will need to update this once initial package version has been published.

#11766 makes this package public so it can be published,